src/GEM_u8g2.cpp

```diff
@@ -282,7 +282,18 @@ void GEM_u8g2::printMenuItemValue(const char* str, int offset, byte startPos) {
   printMenuItemString(str, _menuItemValueLength + offset, startPos);
 }
 
-void GEM_u8g2::printMenuItemValue(GEMItem* menuItemTmp, byte yDraw)
+void GEM_u8g2::printMenuItemValuePrintFunction(const char* str, byte yText, bool printFull){
+  // select menu print function, dependent on if we want to print a value with or without title
+  if (printFull){
+    _u8g2.setCursor(11, yText);
```

Comment:
Where is this 11 came from? 
Is that the count of lines we want to show maximum?

Reply:
You already answered me this question.